### PR TITLE
Add: Software engineering job description

### PR DESCRIPTION
**Pull request by joachimprinzbach**

### CODE REVIEW

File: docs/md/se/jobs/11619.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Was wir erwarten
+
+Dass du Lust auf Community hast. Ein grosses Plus bei der Baloise ist, dass wir uns in einem [agilen Umfeld](https://www.baloise.com/jobs/de/blog/baloise-jobcast/baloise-jobcast-3-autonome-teams-in-der-it-entwicklung.html) in immer neuen Teams zusammenfinden. Das stärkt das Netzwerk und die Zusammenarbeit. Sei neugierig, hab den Antrieb, dich weiterzubilden und nutze die Möglichkeiten. Es gibt regelmässig Fortbildungsangebote auf Konferenzen und Messen, die dich und damit die Baloise weiterbringen. Open X Days, Stages inbegriffen.
+
+Wir sind ein Bereich, in dem man sich gegenseitig unterstützt. Keiner muss allein ins kalte Wasser springen. Auch diese Haltung sollte dir vertraut sein. On top: Offene Menschen in der IT, die gern mit anderen reden. – Schon klar, das ist nicht unser liebstes Hobby als IT'ler, aber wir sind Schnittstelle zum Fachbereich. Heisst, wir müssen unser Know-how zielgruppengerecht verständlich runterbrechen können, um die Anforderungen zu besprechen.
+
+Was du ausserdem als Software Engineer Java mitbringen solltest:
+
+Sehr guter Hochschulabschluss in Informatik.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: use real bullet points

File: docs/md/se/jobs/11619.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Interesse an Software-Architektur.
+
+Ausgezeichnete Deutsch-Kenntnisse. 
+
+## Was du bekommst
+
+- Flexible Arbeitszeiten sind bei der Baloise inklusive. Home Office und Teilzeitoptionen ebenso. 
+
+- Wir sind in verschiedenen Communities organisiert – Developer, Java, Frontend, … – jeder fühlt sich verantwortlich und bringt seinen Teil  ein. Hier geht’s um Know-how Transfer und den Blick über den Tellerrand hinaus.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: We should only mention communities that really exist

File: docs/md/se/jobs/11619.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Agiler Software Engineer Java (m/w) 80-100%
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: Remove "Agiler"

File: docs/md/se/jobs/11619.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Was dich erwartet
+
+Unser [Aufgabenspektrum](https://www.baloise.com/jobs/de/blog/baloise-jobcast/jobcast-it-engineer.html) ist klar. Der Fachbereich kommt mit einer Anforderung, wir besprechen diese gemeinsam und setzen dann entsprechend um.
+
+Klingt mega unspannend, ist es aber nicht, weil du bei uns die kreative Freiheit bekommst, eigene Lösungen zu finden. Wir sind so gar nicht top down, sondern vor allem auf Augenhöhe unterwegs. Du bist die IT-Kompetenz - vom Front- bis ins Backend, Java- bzw. Angular-basiert – und du bringst deine Vorstellung von einer möglichen Umsetzung ein.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: Java and Angular, one is a programming language, the other a SPA framework... 
-> Java and TypeScript, JEE and Angular

File: docs/md/se/jobs/11619.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Warum das Spass macht? Weil wir nah beim Kunden sind. So nah wie man das als IT'ler vielleicht von einem Versicherer nicht erwartet. Wir bekommen direktes Feedback und können unsere Software nutzergerecht optimieren. Du bekommst auch genug Challenge im Team. Wenn du also Spass hast an Austausch, Diskussion und gemeinsamer Entwicklung bist du hier richtig. 
+
+## Was wir erwarten
+
+Dass du Lust auf Community hast. Ein grosses Plus bei der Baloise ist, dass wir uns in einem [agilen Umfeld](https://www.baloise.com/jobs/de/blog/baloise-jobcast/baloise-jobcast-3-autonome-teams-in-der-it-entwicklung.html) in immer neuen Teams zusammenfinden. Das stärkt das Netzwerk und die Zusammenarbeit. Sei neugierig, hab den Antrieb, dich weiterzubilden und nutze die Möglichkeiten. Es gibt regelmässig Fortbildungsangebote auf Konferenzen und Messen, die dich und damit die Baloise weiterbringen. Open X Days, Stages inbegriffen.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: remove "comma" after "Antrieb"

Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: We do not attend "Messen" - besides conferences I would mention meetups, courses / trainings

File: docs/md/se/jobs/11619.md (hunk 3 above)
Comment (MarkusTiede) [reply docs/md/se/jobs/11619.md]: done

File: docs/md/se/jobs/11619.md (hunk 5 above)
Comment (joachimprinzbach) [reply docs/md/se/jobs/11619.md]: What are Open X Days? Maybe mention [OpenSpace](https://en.wikipedia.org/wiki/Open_Space_Technology), what exactly is a Stage? Maybe explain it.

File: docs/md/se/jobs/11619.md (hunk 4 above)
Comment (MarkusTiede) [reply docs/md/se/jobs/11619.md]: done

File: docs/md/se/jobs/11619.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+Wir sind ein Bereich, in dem man sich gegenseitig unterstützt. Keiner muss allein ins kalte Wasser springen. Auch diese Haltung sollte dir vertraut sein. On top: Offene Menschen in der IT, die gern mit anderen reden. – Schon klar, das ist nicht unser liebstes Hobby als IT'ler, aber wir sind Schnittstelle zum Fachbereich. Heisst, wir müssen unser Know-how zielgruppengerecht verständlich runterbrechen können, um die Anforderungen zu besprechen.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: I would completely rework this part - I don't like it at all

File: docs/md/se/jobs/11619.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## Was wir erwarten
+
+Dass du Lust auf Community hast. Ein grosses Plus bei der Baloise ist, dass wir uns in einem [agilen Umfeld](https://www.baloise.com/jobs/de/blog/baloise-jobcast/baloise-jobcast-3-autonome-teams-in-der-it-entwicklung.html) in immer neuen Teams zusammenfinden. Das stärkt das Netzwerk und die Zusammenarbeit. Sei neugierig, hab den Antrieb, dich weiterzubilden und nutze die Möglichkeiten. Es gibt regelmässig Fortbildungsangebote auf Konferenzen und Messen, die dich und damit die Baloise weiterbringen. [Open X Days](https://www.openfriday.org/openfriday-org-deutsch), Stages inbegriffen.
+
+Wir sind ein Bereich, in dem man sich gegenseitig unterstützt. Keiner muss allein ins kalte Wasser springen. Auch diese Haltung sollte dir vertraut sein. On top: Offene Menschen in der IT, die gern mit anderen reden. – Schon klar, das ist nicht unser liebstes Hobby als IT'ler, aber wir sind Schnittstelle zum Fachbereich. Heisst, wir müssen unser Know-how zielgruppengerecht verständlich runterbrechen können, um die Anforderungen zu besprechen.
+
+Was du ausserdem als Software Engineer Java mitbringen solltest:
+ - Sehr guter Hochschulabschluss in Informatik.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: i would align it with the other job: "Ausbildung in Informatik (Hochschule, EFZ o.ä)."

File: docs/md/se/jobs/11619.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+ - Mehrjährige Erfahrung in der Software-Entwicklung sowie überzeugende Leistungsnachweise in erfolgreich abgeschlossenen Software-Projekten.
```
Comment (joachimprinzbach) [inline docs/md/se/jobs/11619.md]: Please remove "überzeugende Leistungsnachweise in erfolgreich abgeschlossenen Software-Projekten." I don't even know, what is expected here.

**Review by MarkusTiede — COMMENTED** [review]
Open X-Day / Friday link added

File: docs/md/se/jobs/11619.md (hunk 6 above)
Comment (MarkusTiede) [reply docs/md/se/jobs/11619.md]: I'd leave it for now - IMHO this has been one of the main re-designs

File: docs/md/se/jobs/11619.md (hunk 7 above)
Comment (MarkusTiede) [reply docs/md/se/jobs/11619.md]: I'd also leave this as is: (imho) there is a pretty clear difference in expectation  when applying with a university degree or without.

**Review by MarkusTiede — APPROVED** [review]

**Review by MarkusTiede — APPROVED** [review]

**Comment by MarkusTiede** [comment]
Delta from the initial contribution: https://github.com/baloise/ccc/compare/be12d1dae0ac8a749296a351b84cd731d01e7f43..6904c2eec1555c0a0d9ed47ec9ca391ef09a8e13

**Comment by MarkusTiede** [comment]
Delta from initial contribution 

https://github.com/baloise/ccc/compare/be12d1dae0ac8a749296a351b84cd731d01e7f43..ce28653?short_path=8f28ce2#diff-8f28ce22a41ae100c607c87d5b1f5e33

**Pull request merged**